AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m

```diff
@@ -0,0 +1,30 @@
+#import "MSTokenResult.h"
+
+@implementation MSTokenResult
+
+@synthesize partition = _partition;
+@synthesize dbAccount = _dbAccount;
+@synthesize dbName = _dbName;
+@synthesize dbCollectionName = _dbCollectionName;
+@synthesize token = _token;
+@synthesize status = _status;
+
+- (instancetype)initWithPartition:(NSString *)partition
+                     andDbAccount:(NSString *)dbAccount
```

Comment:
NIT: remove and prefix e.g. 
dbAccount: :(NSString *)dbAccount
dbName::(NSString *) dbName
...

Reply:
made the change 👍